integration_tests/e2e/overview/goal/reviewUpdateGoal.cy.ts

```diff
@@ -44,10 +44,10 @@ context('Review updated goal', () => {
     const goalReference = '10efc562-be8f-4675-9283-9ede0c19dade'
     cy.signIn()
 
-    cy.visit(`/plan/${prisonNumber}/view/goals#in-progress`)
+    cy.visit(`/plan/${prisonNumber}/view/goals/in-progress-goals`)
     const goalsPage = Page.verifyOnPage(GoalsPage)
     const inProgressGoalsPage = goalsPage.clickInProgressGoalsTab()
-    const updateGoalPage = inProgressGoalsPage.clickUpdateButtonForFirstGoal()
+    const updateGoalPage = inProgressGoalsPage.clickUpdateButtonForGoal(goalReference)
```

Comment:
The reason for these changes is the goal reference in the njk template was being set as the goal number (loop.index) instead of the actual goal reference. It now uses the goal reference (see update to goalsTabContents.njk file below).